legacy-docs/docs/r/fabric_node_member.html.markdown

```diff
@@ -26,24 +59,39 @@ resource "aci_fabric_node_member" "example" {
   node_type   = "unspecified"
   pod_id      = "1"
   role        = "unspecified"
+  commission  = "yes"
 }
+
 ```
+## Schema ##
+
+### Required ###
+
+* `name` (name) - (string) The name of the Fabric Node Member object.
+* `serial` (serial) - (string) Serial Number of the Fabric Node Member.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Fabric Node Member object.
+
+### Optional ###
 
-## Argument Reference
-
-- `serial` - (Required) Serial Number for the new Fabric Node Member.
-- `name` - (Required) Name of Fabric Node member.
-- `annotation` - (Optional) Specifies the annotation of a Fabric Node member.
-- `description` - (Optional) Specifies the description of a Fabric Node member.
-- `ext_pool_id` - (Optional) external pool ID for object Fabric Node member. Default value: "0".
-- `fabric_id` - (Optional) Fabric ID for the new Fabric Node Member. Default value: "1".
-- `name_alias` - (Optional) Name alias for object Fabric Node member.
-- `node_id` - (Optional) Node ID Number for the new Fabric Node Member. Allowed value range: "101" - "4000". Default value: "0".
-- `node_type` - (Optional) Node type for object Fabric Node member.
-  Allowed values: "unspecified", "remote-leaf-wan". Default value: "unspecified".
-- `pod_id` - (Optional) The pod id of the new Fabric Node Member. Allowed value range: "1" - "254". Default value: "1".
-- `role` - (Optional) Role for the new Fabric Node Member. 
+* `annotation` (annotation) - (string) Specifies the annotation of a Fabric Node member.
+* `description` (descr) - (string) Specifies the description of a Fabric Node member.
+* `ext_pool_id` (extPoolId) - (string) External pool ID of the Fabric Node member. Default value: "0".
+* `fabric_id` (fabricId) - (string) Fabric ID of the Fabric Node Member. Default value: "1".
+* `name_alias` (nameAlias) - (string) Name alias of the Fabric Node member.
+* `node_id` (nodeId) -  (string) Node ID Number of the Fabric Node Member.
+  Allowed value range: "101" - "4000". Default value: "0".
+* `node_type` (nodeType) -  (string) Node type of the Fabric Node member.
+  Allowed values are "unspecified", "remote-leaf-wan", "tier-2-leaf", "virtual", "border-gateway". Default value: "unspecified".
```

Comment:
this is done with bullet points if not mistaken in new docs? see below example from epg

```
* `preferred_group_member` (prefGrMemb) - (string) Parameter used to determine whether the EPG is part of the preferred group. Members of this group are allowed to communicate without contracts.
  - Default: `exclude`
  - Valid Values: `exclude`, `include`.

Reply:
changed